Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs

```diff
@@ -71,27 +71,36 @@ public override void OnInspectorGUI()
             EditorGUILayout.PropertyField(handlerType);
 
             EditorGUILayout.Space();
-            showCpuProperties = EditorGUILayout.Foldout(showCpuProperties, "Processor");
+            showCpuProperties = EditorGUILayout.Foldout(showCpuProperties, "Processor", true);
             if (showCpuProperties)
             {
-                EditorGUILayout.PropertyField(showCpu);
-                EditorGUILayout.PropertyField(cpuBuffer);
+                using (new EditorGUI.IndentLevelScope())
```

Comment:
Why not just use the `EditorGUI.indentLevel==` and save the allocations with having a new one?

We're already in brackets for the foldout anyway, so seems rather wasteful.

Reply:
I originally chose the using pattern to prevent forgetting to unindent (defensive coding). The allocations would only occur in the editor, so should not impact on device performance.